frodo.js

```diff
@@ -1,14 +1,14 @@
 /**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?